2_Controllers/test_controller.js

```diff
@@ -46,18 +41,30 @@ const getAllArticles = (request, response, next) => {
 const getCommentsByArticleId = (request, response, next) => {
   const { article_id } = request.params;
   fetchCommentsByArticleId(article_id)
-    // console
-    //   .log(article_id)
     .then((comments) => {
       response.status(200).send({ comments });
     })
     .catch(next);
 };
 
+const postComments = (request, response, next) => {
+  const { article_id } = request.params;
+  const { username, body } = request.body;
+  if (!username || !body) {
```

Comment:
this is not necessary because if username is undefined or the body is undefined then psql will throw an error because undefined will violate the not null constraint and sql will throw an error